test/run_tests.bat

```diff
@@ -5,10 +5,7 @@ set PYTHONPATH=%MYDIR%
 REM Note: shift will not work with %*
 REM ### Get args, Handle Python version ###
 set "_args=%*"
-IF "%1" == "-2" (
-  set PYTHON=python
-  set "_args=%_args:~3%"
-) ELSE IF "%1" == "-3" (
```

Comment:
same

Reply:
I'm not into batch file programming. Sorry! I don't feel competent to fix this file properly.

Reply:
Will try to check this batch file in my spare time how it can be improved